lib/rules/validate-quote-marks.js

```diff
@@ -106,10 +106,16 @@ module.exports.prototype = {
                     return;
                 }
 
+                token.value = token.value.replace(
+                    new RegExp('^' + mark + '(.*)' + mark + '$'),
```

Comment:
It will probably fail on this:

```
var string = "123'456"; // -> '123'456' — syntax error
```

And also. Let's not change token values inside rules. Let's add another assertion to Assert module.